src/server/parser/error.ts

```diff
@@ -90,6 +90,8 @@ export class SyntaxError
 			return 'Expression constitutes an invalid assignment'
 		case ErrorKind.symbolAlreadyDefined:
 			return 'The symbol defined by the expression is already defined in the current scope'
+		default:
```

Comment:
While strictly correct, we'd opted not to do this so the compiler would bark if the programmer forgot to add a new ErrorKind to this switch-case

Reply:
Now the `default` linting error is ignore for this specific case